### DISCUSSION TITLE
Error in Sign-up Page

### DISCUSSION THREAD

**Discussion by ksharshith2004** [code-a2z/jarvis#168] [Q&A] [answered]
i created a issue to create 2048.The game is running alone but together wen i try to run as jarvis.py then i got this error

![Screenshot 2024-10-11 131601](https://github.com/user-attachments/assets/3ca84e46-8215-411b-838b-71abfaf4ec75)

**Comment by Avdhesh-Varshney** [comment, marked as answer]
You have to create a `secrets.toml` file and setup all the variables empty string and fill the necessary variables as shown in `secrets.example.toml` file.
